locations/spiders/enterprise.py

```diff
@@ -26,16 +26,16 @@ def parse_country(self, response):
         if data:
             for d in data:
                 properties = {
-                    'name': d.get('shortTitle'),
-                    'phone': d.get('formattedPhone'),
+                    'name': d.get('name'),
+                    'phone': d.get('formatted_phone'),
                     'website': d.get('url'),
-                    'addr_full': ' '.join(d.get('addressLines')),
+                    'addr_full': d.get('address_lines'),
```

Comment:
I'm gong to flip this back to what it was before, because it puts an array of data in here rather than the expected string.